web/public/js/controllers/session-modal-controller.js

```diff
@@ -38,6 +40,9 @@
     _.each(_.keys($scope.session.tickets), function(ticketId) {
       var ticket = $scope.session.tickets[ticketId];
       $scope.session.tickets[ticketId].remaining = ticket.quantity - ticket.totalApplications;
+      var settings = _.clone($scope.applyForSettings);
+      settings.selectionLimit = ticket.remaining;
```

Comment:
I fear this may generate confusion... Not really sure of the alternatives though...